bbq/compiler/compiler_test.go

```diff
@@ -626,29 +622,35 @@ func TestCompileIfLet(t *testing.T) {
 	functions := comp.ExportFunctions()
 	require.Equal(t, len(program.Functions), len(functions))
 
+	const (
+		xIndex = iota
+		_
+		tempYIndex
+		yIndex
+	)
```

Comment:
Could you please add some comments for these? For example, why the second variable is skipped (`$result` local)